internal/nonkube/client/fs/path_provider.go

```diff
@@ -0,0 +1,19 @@
+package fs
+
+import "fmt"
+
+type PathProvider struct {
+	Namespace string
+}
+
+func (p *PathProvider) getDefaultNamespace() string {
+	return ".local/share/skupper/default/prepared/state"
```

Comment:
Once PR #1641 lands, we can tweak this one and CustomResourceHandler a bit more
as it is taking more shape now, but for now, I would recommend just updating the path,
from: `.local/share/skupper/default/prepared/state` to: `.local/share/skupper/default/sources`

Reply:
Minor, but important. The path above is not the right one.
It is missing the "namespaces", my bad. Here is the correct ones:
`.local/share/skupper/namespaces/default/sources` and
`.local/share/skupper/namespaces/<namespace>/sources`

Reply:
np! fixed 47ab65379ffbb2f3351d7ada4d7bd970e04553cd